src/Backend/Core/Js/ckeditor/plugins/codemirror/theme/3024-day.css

```diff
@@ -0,0 +1,41 @@
+/*

```

Comment:
aren't you adding a whole lot of themes that we don't use here?

Reply:
They're mostly the same as there already were in the previous version.

Reply:
k